src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -140,9 +138,11 @@ async Task<JObject> FindStaticMemberInType(string name, int typeId)
                     {
                         isInitialized = await context.SdbAgent.TypeInitialize(typeId, token);
                     }
-                    var valueRet = await context.SdbAgent.GetFieldValue(typeId, field.Id, token);
-
-                    return await GetValueFromObject(valueRet, token);
+                    var staticFieldValue = await context.SdbAgent.GetFieldValue(typeId, field.Id, name, token);
+                    var valueRet = await GetValueFromObject(staticFieldValue, token);
+                    // we need the full name here
+                    valueRet["className"] = classNameToFind;
+                    return valueRet;
```

Comment:
Why the change? Needs corresponding tests.

Reply:
Without this change this test is failing:
```
DebuggerTests.EvaluateOnCallFrameTests.EvaluateMethodsOnPrimitiveTypesReturningPrimitives [FAIL]
```
for `test.propString.Split('*', 3, System.StringSplitOptions.RemoveEmptyEntries)`
That's why: we are using `className` in `ConvertJSToCSharpLocalVariableAssignment` when casting number to Enum. If we allow className to be "stringSplitOptions", which is the original value returned by Runtime we won't get a valid cast. We needed to somehow get the full name and in this place it's available. This way we are getting "System.StringSplitOptions" in `className`.